hexa/workspaces/schema/mutations.py

```diff
@@ -194,6 +195,9 @@ def resolve_delete_workspace_member(_, info, **kwargs):
         workspace_membership = WorkspaceMembership.objects.get(id=input["membershipId"])
         workspace_membership.delete_if_has_perm(principal=request.user)
 
+        if PipelineRecipient.objects.filter(user=workspace_membership.user).exists():
```

Comment:
Just to ensure we cover all cases, I think it would be better to use the `post_delete` signal instead of doing it here.

https://docs.djangoproject.com/en/dev/ref/signals/#post-delete

Reply:
Okay